Remove embedded ldap related content (#3264)
wso2/docs-is: en/docs/deploy/add-high-availability-for-ldap.md

# Add High Availability for LDAP

> When connecting the WSO2 Identity Server to a LDAP userstore,

Reviewer: ```suggestion
When connecting the WSO2 Identity Server to an LDAP user store,
```
